packages/electron-publish/src/BintrayPublisher.ts

```diff
@@ -72,6 +72,7 @@ export class BintrayPublisher extends HttpPublisher {
             "Content-Length": dataLength,
             "X-Bintray-Override": "1",
             "X-Bintray-Publish": "1",
+            "X-Bintray-Debian-Architecture": arch
```

Comment:
I'll add the other Bintray headers later. I'm passing this one here just to make use of the `arch` variable we're propagating.